### ISSUE TITLE
Wrong help information for "enb server -h" command call

### ISSUE THREAD

**Issue by narqo** [enb-make/enb#101]
`-h` option is doubled in `enb server`'s help:

```shell
  › enb server --help

  Usage: server [options]

  Options:

    -h, --help             output usage information
    -p, --port <port>      socket port [8080]
    -h, --host <host>      socket host [0.0.0.0]
    -s, --socket <socket>  unix socket path
```

```shell
  › enb server -h

  error: option `-h, --host <host>' argument missing
```

**Comment by andrewblond** [comment]
Need remove short `-h` flag of option `host` in `v1.0.0` version.

**Comment by SwinX** [comment]
Closed in favor of #383
